src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
 body {
   margin: 0;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
```

Comment:
You have a 'card' block with a set width and height, which makes it less flexible. Consider making it more responsive by setting max-width instead of width, and let the height be determined by its content.